### DISCUSSION TITLE
The OSPO Forum

### DISCUSSION THREAD

**Discussion by caniszczyk** [todogroup/ospology#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the OSPOlogy community. We hope that you:
  * Ask questions you’re wondering about when it comes to OSPOs
  * Share ideas about how to improve OSPOs
  * Engage with other OSPO community members

We also have real time discussions on the TODO Group Slack http://slack.todogroup.org

**Comment by jsmanrique** [comment]
Good for those that are not allowed to connect to Slack 😉

**Reply by caniszczyk** [reply]
oh my, sometimes banks ;p

**Comment by jlassery** [comment]
Hello everyone, I'm trying to connect via slack, but the link available in the TODO website is not working.

**Reply by anajsana** [reply]
please try [this link](https://join.slack.com/t/thetodogroup/shared_invite/zt-2w71kclgx-JOUB4LTXIuVEKehkJk7V0w)
